src/main/resources/biz_de.properties

```diff
@@ -36,12 +36,12 @@ BizController.deleteProcess = Löschvorgang
 BizController.deleteProcessTitle = Löschen von '${entity}'
 BizController.deletingInBackground = Das Datenobjekt enthält einige verknüpfte Datenobjekte und wird deshalb im Hintergrund gelöscht.
 BizController.illegalArgument = Fehlerhafte Eingabe. Bitte korrigieren Sie die markierten Felder
-BizController.invalidReference = Eine Beziehung zwischen den gegebenen Datenobjekten ist inkonsistent.
+BizController.invalidReference = Im Objekt '${child}' wurde das Übergeordnete Objekt '${parent}' erwartet, aber es wurde '${actual}' gefunden.
```

Comment:
```suggestion
BizController.invalidReference = Im Objekt '${child}' wurde das referenzierte Objekt '${parent}' erwartet, aber es wurde '${actual}' gefunden.
```

Reply:
Can we also include the type in this error message?

Reply:
$parent , $child are the unique-name of the respective entity, which is something like 'MongoTenant-123' , i did this as i found the error message with 'id' of 'type' too long.

Reply:
sounds good